src/display.c

```diff
@@ -30,6 +30,7 @@
 #include <time.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <omp.h>
```

Comment:
I think we need a test program to compare the performance before and after OpenMP is enabled.

Reply:
I came up with a suitable test:

- Set window size to 1920x1080
- Create 21384 widgets, they're all 10x10 in size
- Render 1000 frames
- For each frame, set the background color of each widget randomly

This is an optional task, if you are interested in it, you can try do it.